streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -335,12 +335,23 @@
 
     /**
      * Count number of messages of this stream by key without a window basis, and hence
+     * return a ever updating counting table.
+     * This method uses {@link org.apache.kafka.common.serialization.LongSerializer LongSerializer}
```

Comment:
Typically, I would prefix such an implementation detail with, doh, "Implementation detail: ....".

This highlights that the information is not part of the API contract, but rather a FYI.